plugins/rapid7_insightvm/unit_test/test_delete_asset.py

```diff
@@ -1,37 +1,41 @@
 import sys
 import os
 from unittest import TestCase
-from komand_rapid7_insightvm.connection.connection import Connection
+from unittest.mock import patch
+from unit_test.util import Util
 from komand_rapid7_insightvm.actions.delete_asset import DeleteAsset
-import json
-import logging
+from komand_rapid7_insightvm.actions.delete_asset.schema import Input
+from parameterized import parameterized
+from insightconnect_plugin_runtime.exceptions import PluginException
 
 sys.path.append(os.path.abspath("../"))
```

Comment:
You should move 
`sys.path.append(os.path.abspath("../"))` 
above 
`from unit_test.util import Util` 
as this import is using the relative path added by sys.path

Otherwise tests runned from unit_test directory are failing.

This should be changed in all unit_tests